### PR TITLE
FOUR-19004: Create a NEW VIEW for cases Part Counters

### PR DESCRIPTION
**Pull request by henryjonathanquispe**
## Feature
Add new counters in case details

## Related Tickets & Packages
- https://processmaker.atlassian.net/browse/FOUR-19004

## Code Review Checklist
- [ ] I have pulled this code locally and tested it on my instance, along with any associated packages.
- [ ] This code adheres to [ProcessMaker Coding Guidelines](https://github.com/ProcessMaker/processmaker/wiki/Coding-Guidelines).
- [ ] This code includes a unit test or an E2E test that tests its functionality, or is covered by an existing test.
- [ ] This solution fixes the bug reported in the original ticket.
- [ ] This solution does not alter the expected output of a component in a way that would break existing Processes.
- [ ] This solution does not implement any breaking changes that would invalidate documentation or cause existing Processes to fail.
- [ ] This solution has been tested with enterprise packages that rely on its functionality and does not introduce bugs in those packages.
- [ ] This code does not duplicate functionality that already exists in the framework or in ProcessMaker.
- [ ] This ticket conforms to the PRD associated with this part of ProcessMaker.

ci:next
ci:deploy

### CODE REVIEW

File: resources/jscomposition/cases/casesMain/utils/counters.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    const counters = [
+        {
+            header: "My cases",
+            body:data.myCases.toString(),
+            color:"amber",
+            icon:"fa-regular fa-user",
+            url:'/cases-main/my-cases'
+        },
+        {
+            header: "In progress",
```
Comment (pmPaulis) [inline resources/jscomposition/cases/casesMain/utils/counters.js:13]: We need to use a translation for this label

File: resources/jscomposition/cases/casesMain/utils/counters.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+            body:data.inProgress.toString(),
+            color:"green",
+            icon:"fa-solid fa-list",
+            url:'/cases-main/in-progress'
+        },
+        {
+            header: "Completed",
```
Comment (pmPaulis) [inline resources/jscomposition/cases/casesMain/utils/counters.js:20]: We need to use a translation for this label

File: resources/jscomposition/cases/casesMain/utils/counters.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+            body:data.completed.toString(),
+            color:"blue",
+            icon:"fa-regular fa-circle-check",
+            url:'/cases-main/completed'
+        },
+        {
+            header: "All cases",
```
Comment (pmPaulis) [inline resources/jscomposition/cases/casesMain/utils/counters.js:27]: We need to use a translation for this label

File: resources/jscomposition/cases/casesMain/utils/counters.js [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+            body:data.allCases.toString(),
+            color:"purple",
+            icon:"fa-regular fa-clipboard",
+            url:'/cases-main/all-cases'
+        },
+        {
+            header: "All requests",
```
Comment (pmPaulis) [inline resources/jscomposition/cases/casesMain/utils/counters.js:34]: We need to use a translation for this label

File: ProcessMaker/Http/Controllers/CasesController.php [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+use ProcessMaker\Package\PackageComments\PackageServiceProvider;
+use ProcessMaker\ProcessTranslations\ProcessTranslation;
+use ProcessMaker\RetryProcessRequest;
+use ProcessMaker\Traits\HasControllerAddons;
+use ProcessMaker\Traits\SearchAutocompleteTrait;
+use Spatie\MediaLibrary\MediaCollections\Models\Media;
+
+class CasesController extends Controller
+{
+        /**
```
Comment (pmPaulis) [inline ProcessMaker/Http/Controllers/CasesController.php:34]: Please update this method like
```
    /**
     * Get the list of requests.
     *
     * @return \Illuminate\View\View|\Illuminate\Contracts\View
     */
    public function index($type = null)
    {
       // This is a temporary API the engine team will provide the new
        return view('cases.casesMain');
    }
```

File: ProcessMaker/Http/Controllers/CasesController.php [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+     * Get the list of requests.
+     *
+     * @return \Illuminate\View\View|\Illuminate\Contracts\View
+     */
+    public function index($type = null)
+    {
+
```
Comment (pmPaulis) [inline ProcessMaker/Http/Controllers/CasesController.php:41]: Pleasee add a comment:
`// This is a temporary API the engine team will provide the new`

**Review by pmPaulis — CHANGES_REQUESTED** [review]

File: resources/jscomposition/cases/casesMain/routes.js
Comment (fagubla) [inline resources/jscomposition/cases/casesMain/routes.js:1]: use a correct indentation

File: resources/jscomposition/cases/casesMain/config/columns.js
Comment (fagubla) [inline resources/jscomposition/cases/casesMain/config/columns.js:1]: missing translation for header name

File: resources/views/cases/casesMain.blade.php [hunk 7]
```diff
@@ -0,0 +1,24 @@
+@extends('layouts.layout')
+
+<link href="{{ mix('css/tailwind.css') }}" rel="stylesheet">
```
Comment (fagubla) [inline resources/views/cases/casesMain.blade.php:3]: this line needs to be removed, in other PR this line is added in the layout.blade

**Review by fagubla — CHANGES_REQUESTED** [review]

**Review by fagubla — APPROVED** [review]

**Pull request merged**